backend/app/Models/Publication.php

```diff
@@ -4,18 +4,43 @@
 namespace App\Models;
 
 use Illuminate\Database\Eloquent\Factories\HasFactory;
-use Illuminate\Database\Eloquent\Model;
 use Illuminate\Database\Eloquent\Relations\BelongsToMany;
 use Illuminate\Database\Eloquent\Relations\HasMany;
 
-class Publication extends Model
+class Publication extends BaseModel
 {
     use HasFactory;
 
     protected $fillable = [
         'name',
     ];
 
+    protected $rules = [
+        'name' => 'max:256|unique:publications,name|required',
```

Comment:
Bc when it was on the schema and not the model it was pulling double duty. If the request was an update it may not be there, but if it is, it needed to be filled. The requirement to for it to be present on create was fulfilled by the ! On the create inputs name field. 

(To that effect, one could totally create a entity with an empty name by creating and save if the model directly with tinker)

With it on the model, the require is satisfied by the existing non empty name on update. Tinker should no longer allow creating a model with an empty name. 

The long and short, in the future when we implement plugins that may interact with the underlying models rather than the graph they will require a name. 

I plan to clean up some of our other validation in the future similarly.